### ISSUE TITLE
[Bug]: When trying to scaffold using a taqueria binary built with deno version 1.24.0 it will give error Busy: Resource is unavailable because it is in use by a promise

### ISSUE THREAD

**Issue by alexzbusko** [ecadlabs/taqueria#1050]
### What happened?

When running the base `taq scaffold` command on a machine running deno 1.24.0 it is returning the error in the log output

### What should have happened?

It should have installed the taco-shop scaffold with no issues

### Steps to reproduce the bug

Steps to reproduce the error:

1. Install deno 1.24.0. with `deno upgrade --version 1.24.0`
3. Build taqueria with `npm run build-all`
4. Run the command `taq scaffold`

### Relevant log output

```shell
$ taq scaffold

 Scaffolding 🛠 
 into: /home/alexander/taqueria/taqueria-taco-shop
 from: https://github.com/ecadlabs/taqueria-scaffold-taco-shop.git 

Cloning into '/home/alexander/taqueria/taqueria-taco-shop'...

 Initializing Project...
    ✓ Remove Git directory
    ✓ Install plugins
    ✓ Run scaffold post-init script
    ✓ Project Taq'ified 

error: Uncaught (in promise) Busy: Resource is unavailable because it is in use by a promise
    at opSync (deno:core/01_core.js:170:12)
    at Console.<anonymous> (deno:runtime/js/99_main.js:470:40)
    at console.log (deno:ext/console/02_console.js:1914:22)
    at EncaseP$res (https://cdn.jsdelivr.net/gh/fluture-js/Fluture@14.0.0/dist/module.js:1270:7)
```

### Screenshots and images

Screenshot and images

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sinapsist** [comment]
![image](https://user-images.githubusercontent.com/10706087/181569975-822086bf-47c7-4515-a315-7b50170e37f5.png)

**Comment by hu3man** [comment]
@mweichert, if you would like to reproduce this for investigation, you simply need to update deno to v1.24.0 or later and try to build taq

I'm going to consider this Done as we fixed the issue by pinning to v1.23.4 in ci
